scripts/extractEdition.py

```diff
@@ -0,0 +1,46 @@
+from elasticsearch_dsl import Q
+from managers import DBManager
+
+from model import Edition
+
+#Extracting the edition number from the edition_statement column of the DRB database to fill out the edition column
+def main():
+
+    dbManager = DBManager(
+        user='',
+        pswd='',
+        host='',
+        port='',
+        db=''
+    )
+
+    dbManager.generateEngine()
+
+    dbManager.createSession()
+
+    for edit in dbManager.session.query(Edition) \
+        .filter(Edition.edit_statement != None and Edition.languages[0]['language'] == 'English'):
+            editStatement = edit.edition_statement
+            editDict = {'first': 1, 'second': 2, 'third': 3, 'fourth': 4, 'fifth': 5, \
+                        'sixth': 6, 'seventh': 7, 'eigth': 8, 'ninth': 9, 'tenth': 10}
```

Comment:
re: question in the PR, we _could_ create mappings like this for common terms in French, German, Japanese, and other languages that commonly show up with edition statements. We could probably get the stats on these languages with some creative SQL and then write up the crosswalks.

(this is if we can't figure out an easy translation tool)